bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseImportOptions.java

```diff
@@ -0,0 +1,23 @@
+package com.google.cloud.bigtable.dataflow;
```

Comment:
Added license comment to the two new files I added. The two borrowed files have their original license by dataflow intact.